plutus-core/exe/Main.hs

```diff
@@ -6,43 +6,50 @@
 
 module Main (main) where
 
-import qualified Language.PlutusCore                               as PLC
-import qualified Language.PlutusCore.CBOR                          as PLC
-import qualified Language.PlutusCore.Evaluation.Machine.Cek        as PLC
-import qualified Language.PlutusCore.Evaluation.Machine.Ck         as PLC
-import qualified Language.PlutusCore.Generators                    as Gen
-import qualified Language.PlutusCore.Generators.Interesting        as Gen
-import qualified Language.PlutusCore.Generators.Test               as Gen
-import qualified Language.PlutusCore.Pretty                        as PP
-import qualified Language.PlutusCore.StdLib.Data.Bool              as StdLib
-import qualified Language.PlutusCore.StdLib.Data.ChurchNat         as StdLib
-import qualified Language.PlutusCore.StdLib.Data.Integer           as StdLib
-import qualified Language.PlutusCore.StdLib.Data.Unit              as StdLib
-import qualified Language.UntypedPlutusCore                        as UPLC
-import qualified Language.UntypedPlutusCore.Evaluation.Machine.Cek as UPLC
+import qualified Language.PlutusCore                                as PLC
+import qualified Language.PlutusCore.CBOR                           as PLC
+import qualified Language.PlutusCore.Evaluation.Machine.Cek         as TCek
+import qualified Language.PlutusCore.Evaluation.Machine.Ck          as Ck
+import           Language.PlutusCore.Evaluation.Machine.ExBudgeting (ExBudget (..), ExBudgetMode (..),
+                                                                     ExBudgetState (..), ExRestrictingBudget (..),
+                                                                     ExTally (..), Hashable)
+import           Language.PlutusCore.Evaluation.Machine.ExMemory    (ExCPU (..), ExMemory (..))
+import qualified Language.PlutusCore.Generators                     as Gen
+import qualified Language.PlutusCore.Generators.Interesting         as Gen
+import qualified Language.PlutusCore.Generators.Test                as Gen
+import qualified Language.PlutusCore.Pretty                         as PP
+import qualified Language.PlutusCore.StdLib.Data.Bool               as StdLib
+import qualified Language.PlutusCore.StdLib.Data.ChurchNat          as StdLib
+import qualified Language.PlutusCore.StdLib.Data.Integer            as StdLib
+import qualified Language.PlutusCore.StdLib.Data.Unit               as StdLib
+import qualified Language.UntypedPlutusCore                         as UPLC
+import qualified Language.UntypedPlutusCore.Evaluation.Machine.Cek  as Cek
```

Comment:
I changed this because everything was called PLC or UPLC and It was confusing me.